doc/tests/empty.rst

```diff
@@ -1,11 +1,20 @@
 ``empty``
 =========
 
-``empty`` checks if a variable is empty:
+``empty`` checks if a variable is
+ - an empty string
+ - an empty array
+ - exactly ``false``
+ - exactly ``null``
+ - an instance of Countable_ and has a count_ of zero
```

Comment:
This last sentence should be converted into something more Twig-related like: `an empty array or an empty hash`